package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "google-container-slack",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Slack integration for Google Cloud Container Builder, using Google Cloud Functions",
   "main": "index.js",
   "scripts": {
-    "test": "node_modules/.bin/mocha --reporter spec",
+    "test": "nyc mocha",
```

Comment:
That would only work if you installed `nyc` globally. Please roll back.

Reply:
Ah true, thanks.
I've rolled back to using the `node_modules/.bin/` also included the local `nyc`, is that ok?